lib/pages/enter_page.dart

```diff
@@ -156,26 +157,56 @@ class _EnterPageState extends State<EnterPage> {
     );
   }
 
-  void onPressedButton() {
+  void onPressedButton() async{
 
-    if (_formKey.currentState!.validate()) {
-      String userBD = "fjuliaaf";
-      String passwordBD = "12345";
+    // if (_formKey.currentState!.validate()) {
+    //   String userBD = "fjuliaaf";
+    //   String passwordBD = "12345";
+
+    //   String user = userController.text;
+    //   String pwd = passwordController.text;
 
+    //   if (userBD == user && passwordBD == pwd) {
+    //     Navigator.pushAndRemoveUntil(
+    //       context,
+    //       MaterialPageRoute(
+    //         builder: (context) {
+    //           return const HomePage();
+    //          },
+    //       ),
+    //       (Route<dynamic> route) => false,
+    //     );
+    //   } 
+    // }
```

Comment:
Se não for utilizar ou não for pretender falar sobre isso na apresentação do projeto, acho interessante remover os comentários.